apps/cron/src/tasks/processBuilderActivity/recordCommit.ts

```diff
@@ -187,6 +188,16 @@ export async function recordCommit({ commit, season }: { commit: RequiredCommitF
           });
         }
 
+        try {
+          await completeQuest(githubUser.builderId, 'first-repo-contribution');
+          await completeQuest(githubUser.builderId, 'score-first-commit');
+        } catch (error) {
+          log.error('Error completing quest for commit', {
+            error,
+            builderId: githubUser.builderId
```

Comment:
'userId' not builderId. Datadog has an index on userId and it's easier to search by one property